core/src/main/scala/cmt/Error.scala

```diff
@@ -58,19 +60,4 @@ extension (self: caseapp.core.Error)
     extractOther(self)
   }
 
-extension (cmtError: CmtError)
-  def prettyPrint: String =
-    cmtError match {
-      case RequiredOptionIsMissing(OptionName(value)) =>
-        s"Missing option: $value"
-      case FailedToValidateArgument(option, reasons) =>
-        s"""Option ${option.value}:
-           |${reasons.mkString("   ", "\n   ", "\n")}""".stripMargin
-      case FailedToExecuteCommand(reason) =>
-        s"""Failed to execute command:
-           |  ${toConsoleYellow(reason.message)}
-           |""".stripMargin
-      case FailedToValidateCommandOptions(reasons) =>
-        s"""To be implemented:
-           |${reasons}""".stripMargin
-    }
+extension (cmtError: CmtError) def prettyPrint: String = cmtError.toDisplayString
```

Comment:
so `prettyPrint` is, effectively, just an alias for `toDisplayString`?

should we just change the name of `toDisplayString` to `prettyPrint`?